src/main/java/org/folio/entlinks/controller/converter/AuthoritySourceFileMapper.java

```diff
@@ -47,8 +47,8 @@ public interface AuthoritySourceFileMapper {
   AuthoritySourceFileDto toDto(AuthoritySourceFile authoritySourceFile);
 
   @Mapping(target = "authoritySourceFileCodes",
-           expression = "java(toEntityCodes(authoritySourceFileDto.getCodes()))")
-  @Mapping(target = "source", expression = "java(toSource(authoritySourceFileDto.getSource()))")
+           expression = "java(toEntityCodes(authoritySourceFileDto, authoritySourceFile))")
+  @Mapping(target = "hridStartNumber", source = "hridManagement.startNumber")
   @BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
   AuthoritySourceFile partialUpdate(AuthoritySourceFilePatchDto authoritySourceFileDto,
```

Comment:
is ```_version``` from dto also going to be mapped to entity's ```version```?

Reply:
In Java generated file it's `version`  for dto also, same as for Authority, so no need for explicit mapping